src/version.c

```diff
@@ -176,10 +176,9 @@ static int get_version_from_url(char *url)
 	if (globals.sigcheck) {
 		sig_verified = verify_signature(url, &tmp_version);
 	} else {
-		error("FAILED TO VERIFY SIGNATURE OF %s. Operation proceeding due to \n"
-		      " --nosigcheck, but system security may be compromised\n",
-		      url);
-		journal_log_error("swupd security notice:  --nosigcheck used to bypass version signature");
+		warn("The --nosigcheck flag was used, THE SIGNATURE OF %s WILL NOT BE VERIFIED\n", url);
```

Comment:
Maybe we should stop using Capitalization

Reply:
yeah, I don't like the capitalization either, I didn't want to change it to lower case just so I was more conservative in my change, I can certainly change it to lowercase y @otaviobp is ok with that.

Reply:
I'll change that to normal lower case

Reply:
after some discussion, we decided to leave these messages as upper case